docker/local/Dockerfile

```diff
@@ -19,14 +19,16 @@ RUN echo "deb http://httpredir.debian.org/debian jessie main" > /etc/apt/sources
  && apt-get install -y --no-install-recommends \
 	bash-completion \
 	curl \
+	dkms \
```

Comment:
If you'd like to add them as well in this PR, you are welcome! Otherwise it's fine as it is.